lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -260,26 +306,88 @@ public static Domain getDomain(
         return Domain.create(ValueSet.all(type), hasNullValue);
     }
 
-    private static Optional<ParquetIntegerStatistics> toParquetIntegerStatistics(Statistics<?> statistics, ParquetDataSourceId id, String column)
+    @VisibleForTesting
+    public Domain getDomain(Type type, ColumnIndex columnIndex, ParquetDataSourceId id, RichColumnDescriptor descriptor)
```

Comment:
Move this method below so that the diff lines up better. Currently, it's trying to match `toParquetIntegerStatistics` with `getDomain` instead of `toParquetLongStatistics`.